CONTRIBUTING.md

```
@@ -23,18 +23,18 @@ Please take a moment to review these guidelines before making a contribution. Fo
 
 ## 1. Getting Started
 
-*   **Fork the Repository:** Start by forking the main Kmtemplate repository to your GitHub account.
+*   **Fork the Repository:** Start by forking the main Synapse repository to your GitHub account.
 *   **Clone Your Fork:** Clone your forked repository to your local machine.
-*   *   **Set Up Upstream Remote:** Add the original Kmtemplate repository as an "upstream" remote. This allows you to sync your fork with the latest changes from the main project.
+*   *   **Set Up Upstream Remote:** Add the original Synapse repository as an "upstream" remote. This allows you to sync your fork with the latest changes from the main project.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)